features/detect_anr.feature

```diff
@@ -33,6 +61,15 @@ Scenario: Sleeping the main thread with pending touch events
     And I clear any error dialogue
     Then I should receive no requests
 
+@skip_android_8_1
+Scenario: Sleeping the native main thread with pending touch events with ANR disabled
+    When I run "CXXAnrWithNdk_AnrDisabledScenario"
+    And I wait for 2 seconds
+    And I tap the screen 3 times
+    And I wait for 4 seconds
+    And I clear any error dialogue
+    Then I should receive no requests
```

Comment:
This, along with `CXXAnrWithNdk_AnrDisabledScenario`, doesn't trigger an ANR so the test will always pass. As a general rule mazerunner scenarios work best when they always send a request, as otherwise the fixture could crash and we'd never know. I'd suggest calling `Bugsnag.notify` after the ANR has been triggered to achieve this.